french/manger/faim.md

```diff
@@ -4,3 +4,5 @@ commencer la journée ! ;)
 A force de manger vous faites un trou dans le mur
 
 [Je sors](../sortie/chercher-la-sortie.md)
+[Je me faufille par le trou](trou/trou.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.